roles/development/tasks/main.yml

```diff
@@ -20,7 +20,3 @@
     - include_tasks: python.yml
     - include_tasks: vim.yml
   ignore_errors: true
```

Comment:
I guess we can remove L11-L14 too then

Reply:
I kept 11-14 and added a fail check, then just used my_settings_dir for checks elsewhere.